docs/resource_dependencies.md

```diff
@@ -96,6 +96,15 @@ Kubernetes has a built-in notion of
 [readiness](https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/#define-readiness-probes).
 Tilt will use the built-in readiness probe when it's available.
 
+For any Kubernetes object type, including built-ins and CRDs, that creates pods,
+Tilt will consider it ready when the pod is running and all of its containers
+are ready. The exception to this rule is jobs, which are considered ready by Tilt when
+the job has completed.
+
+If an object does not create pods, there isn't a way for Tilt to determine readiness.
+Instead, you'll want to configure the resource to let Tilt know that it shouldn't
+wait to see pods to determine readiness.
```

Comment:
```suggestion
Instead, you'll want to configure the resource to let Tilt know that it shouldn't
try to use pods to determine readiness. Resources configured to ignore pods
for readiness will appear as ready in the Tilt UI as soon as they are applied to
the Kubernetes cluster.
```

This explanation is great!

You don't have to take my suggestion as-is, but I think it might be useful to have another sentence explaining what will happen after making this change.

Reply:
Could you link to documentation that explains how I can ignore pods for readiness, here?

Reply:
sounds good! thank you both. 

there's a code example right below this paragraph, but i'll also link to the api docs so it's super clear.